src/amp-story-player/amp-story-player-impl.js

```diff
@@ -527,6 +544,12 @@ export class AmpStoryPlayer {
             );
           });
 
+          const customUIConfig = this.getCustomUI_();
+
+          if (customUIConfig) {
+            messaging.sendRequest('setStoryCustomUI', customUIConfig, false);
```

Comment:
Do we really need to pass all the configuration? Can you link to your design doc / tickets for this new viewer messaging?

If the goal is to inject new controls we should have a more accurate message name and API that better reflect our goal. `customUI` could be anything

Reply:
Just created the ticket for the viewer messaging: https://github.com/ampproject/amphtml/issues/30588